dev-env.yaml

```diff
@@ -18,6 +18,9 @@ services:
       - "8080:8080"
     volumes:
       - ./frontend:/app
+    depends_on:
+      - api
+      - ocr
```

Comment:
Would it makes sense to add `depends_on:  - db`

Reply:
Maybe? But I would argue that the frontend doesn't know anything about the DB.

Reply:
I sort of imagined it as DB then API then, OCR then frontend but I guess it is ok to keep the logic separate.